app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/homescreen/HomeScreen.kt

```diff
@@ -0,0 +1,567 @@
+        "series2",
+        "series3",
+        "series4",
+        "series5",
+        "series6",
+    )
+
+private val mockAnime =
+    listOf(
+        "anime1",
+        "anime2",
+        "anime3",
+        "anime4",
+        "anime5",
+        "anime6",
+    )
+
+private val mockSoaps =
+    listOf(
+        "soaps1",
+        "soaps2",
+        "soaps3",
+        "soaps4",
+        "soaps5",
+        "soaps6",
+    )
+
+private val mockSpecials =
+    listOf(
+        "specials1",
+        "specials2",
+        "specials3",
+        "specials4",
+        "specials5",
+        "specials6",
+    )
+
+@Composable
+fun HomeScreen(modifier: Modifier = Modifier) {
+    var currentCategory by rememberSaveable {
+        mutableStateOf(ContentType.MOVIE)
+    }
+
+    val currentItems =
+        when (currentCategory) {
+            ContentType.MOVIE -> mockMovies
+            ContentType.SERIES -> mockSeries
+            ContentType.ANIME -> mockAnime
+            ContentType.SOAPS -> mockSoaps
+            ContentType.SPECIALS -> mockSpecials
+        }
+
+    val brush =
+        Brush.verticalGradient(
+            colorStops =
+            arrayOf(
+                0.0f to lightPurple,
+                0.6f to black_100,
+            ),
+        )
+
+    Box(
+        modifier =
+        modifier
+            .fillMaxSize()
+            .background(
+                brush = brush,
+            ),
+    ) {
+        HomeScreenHeadline(
+            modifier = Modifier.offset(y = dimensionResource(R.dimen.home_screen_headline_y_offset)),
+        )
+        HomeScreenSearchBar(
+            modifier = Modifier.zIndex(3f),
+        )
+
+        HomeScreenChipGroup(
+            onChipClick = { category ->
+                currentCategory = category
+            },
+            categories = ContentType.entries.toList(),
+            modifier =
+            Modifier
+                .offset(y = dimensionResource(R.dimen.home_screen_chip_group_y_offset))
+                .zIndex(2f)
+                .fillMaxWidth(),
+        )
+        HomeScreenContent(
+            items = currentItems,
+            popularItems = currentItems,
+            modifier =
+            Modifier
+                .offset(y = dimensionResource(R.dimen.home_screen_content_y_offset))
+                .zIndex(1f)
+                .fillMaxSize(),
+        )
+    }
+}
+
+@Composable
+private fun HomeScreenHeadline(modifier: Modifier = Modifier) {
+    Column(
+        modifier = modifier,
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.Top,
+            modifier =
+            Modifier
+                .fillMaxWidth()
+                .padding(
+                    horizontal = dimensionResource(R.dimen.home_screen_horizontal_padding),
+                ),
+        ) {
+            Text(
+                text = "What do you want to\nwatch today?",
+                color = white,
```

Comment:
Pass the color from the theme

Reply:
I removed color because I styled this text